docs/source/ContributionGuide.rst

```diff
@@ -242,6 +242,38 @@ approved by your employer) and you grant a full, complete, irrevocable
 copyright license to all users and developers of the Avocado project,
 present and future, pursuant to the license of the project.
 
+Code Review
+-----------
+
+Every single Pull Request in Avocado has to be reviewed by at least one
+other developer. All members of the core team have permission to merge
+a Pull Request, but there are some conditions that have to be fulfilled
+before merging the code:
+
+- Pull Request has to pass the CI tests.
+- One 'Approved' code review should be given.
+- No explicit disapproval should be present.
+
+Pull Requests failing in CI will not be merged, and reviews won't be
+given to them until all the problems are sorted out. If you want help
+to debug the issues with the CI, please reach the developers and/or the
+community using the available communication channels.
+
+While reviewing the code, one should:
+
+- Verify that the code is sound and clean.
+- Run the highest level of selftests (see `make`).
```

Comment:
The highest level is what we currently call `make check-full`.  Since we're *not* running that under CI, I think we should indeed be running it locally.  TBH, I have been running `make check`, but I support changing that.